data/styles/clear/include/Icons.mapcss

```diff
@@ -68,6 +68,7 @@ node[amenity=waste_basket],
 node[amenity=waste_disposal],
 area[amenity=waste_disposal],
 node[man_made=survey_point],
+area[man_made=survey_point],
```

Comment:
according to OSM wiki it can't be an area, only a node

Reply:
fixed